integration/test/test_omp_outer_loop.cpp

```diff
@@ -21,6 +22,7 @@ int main(int argc, char **argv)
     int comm_rank = -1;
     int comm_size = -1;
     err = MPI_Init(&argc, &argv);
+    sleep(5);
```

Comment:
Comment on why.  Please do this in all the places that have a newly added sleep 5.